docs/src/simulation.md

```diff
@@ -0,0 +1,84 @@
+# ALBA Parameters Simulation
+
+ALBA provides strong security guarantees about the capability to generate a proof given some number of items \\(S_p\\):
+
+* a honest prover with more than \\(n_p\\) items has a probability of
+  failing to generate a proof lower than \\(2^{-128}\\),
+* symetrically, a (dishonest) prover with less than \\(n_f\\) items
+  has probability of succeeding to generate a proof lower than
+  \\(2^{-128}\\).
+
+This page provides a simple and user-friendly way of estimating how
+various ALBA parameters impact the size of the proof and probability
+of having a proof, given a number of items \\(S_p\\) varying between
+the two parameters' bounds. It is expected to be a quick way to
+estimate what would be "good" values for those parameters in a
+specific context, depending on security requirements and expected
+adversarial share.
+
+One can define:
+
+* \\(n_p\\): The size of the _honest set_ of items.
+* \\(n_f\\): The size of the _faulty set_ of items. \\(n_f\\) must be lower than \\(n_p\\).
+* Item size: The size of a single item, this is used to compute the full size of a proof.
+* \\(S_p\\): The number of items actually available to an adversary, or put differently, how much adversarial stake should we expect.
```

Comment:
> $S_p$: The number of items actually available to an adversary, or put differently, how much adversarial stake should we expect.

Hmm, depending on the context and parameters, the adversary might still be able to do weird things even if they created none of the $S_p$ many items; for example, in the context of Peras, citing from https://github.com/cardano-scaling/alba/discussions/17:

> Suppose that honest participants only attempt to create an ALBA proof when they have at least some minimum number of votes. Suppose that an adversary is eligible for sufficiently many votes such that the honest participants do not reach this threshold. Then the adversary might still be able to create a valid ALBA proof, causing confusion at the very least.

Phrased differently: An adversary might be able to create a certificate signalling a quorum despite the quorum having been missed, using only honest votes.

Reply:
I will simplify the sentence, because what $S_p$ is in this context is just the number of items one has at their disposal to create a certificate and there's not really any notion of an adversary anymore.